nexterp_stock_date/models/stock.py

```diff
@@ -1,28 +1,46 @@
-# Copyright (C) 2021 NextERP Romania SRL
+# Copyright (C) 2021-2022 NextERP Romania SRL
 # License OPL-1.0 or later
 # (https://www.odoo.com/documentation/user/14.0/legal/licenses/licenses.html#).
 
-from datetime import datetime
-
-from odoo import api, fields, models
+from datetime import timedelta
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
 
 
 class StockPicking(models.Model):
     _inherit = "stock.picking"
 
-    date = fields.Datetime('Processing Date')
+    accounting_date = fields.Datetime(
+        "Accounting Date",
+        copy=False,
+        help="If this field is set, the svl and accounting entiries will "
+        "have this date, If not will have the today date as it should be",
+    )
 
-    @api.depends('move_lines.state', 'move_lines.date', 'move_type')
-    def _compute_scheduled_date(self):
-        super()._compute_scheduled_date()
-        for picking in self:
-            picking.date = picking.scheduled_date
+    date_done_effective = fields.Datetime(
```

Comment:
Scoate te rog asta, ca si istoric se vede in mail chatter cand a fost schimbat in done...sunt prea multe date acolo.

Reply:
date_done e folosit in stock_enterprise  si deci este acelasi ca accounting_date